geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneClientSecurityDUnitTest.java

```diff
@@ -87,24 +98,41 @@ private void startClient(String userName, int serverPort) {
   }
 
   private void executeTextSearch(boolean expectAuthorizationError, String expectedResponse)
-      throws LuceneQueryException {
+      throws LuceneQueryException, InterruptedException {
     LuceneService service = LuceneServiceProvider.get(getCache());
     LuceneQuery<Integer, TestObject> query =
-        service.createLuceneQueryFactory().create(INDEX_NAME, REGION_NAME, "test", "text");
+        service.createLuceneQueryFactory().create(INDEX_NAME, REGION_NAME, "hello", "field1");
+    try {
+      service.waitUntilFlushed(INDEX_NAME, REGION_NAME, 5, TimeUnit.MINUTES);
```

Comment:
why need 5 minutes to prove failure? Will it cause the whole precheckin to be 5 minutes longer?

Reply:
Actually, we shouldn't wait the whole time - if the user doesn't have access the call will fail immediately. I just put 5 minutes because waitUntilFlushed requires a timeout.